.eslintrc.js

```diff
@@ -19,6 +19,12 @@ module.exports = {
         "no-console": 0,
       },
     },
+    {
+      files: ["web/themes/new_weather_theme/assets/js/components/**/*.js"],
+      rules: {
+        "class-methods-use-this": 0,
```

Comment:
Yes. It's a terrible rule, because you can't actually use a static method without referencing the class by name or by constructor, which is clunkkkkkky